.idea/codeStyles/Project.xml

```diff
@@ -1,19 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <option name="LINE_SEPARATOR" value="&#10;" />
```

Comment:
Why is the change to the code styles needed?

Reply:
The changes from this PR still come from the other branch, which included the extension-sdk via git submodules, so we needed to have the same code style in both projects.
Nevertheless we will need a uniform code style as using the project in a composite build in standalone should result in the same formatting.
This code style config was already applied to the hivemq-extension-sdk and hivemq-mqtt-client.

Reply:
We really need a unified style. It very annoying if this keeps changing.

Reply:
This change is here to establish a uniform code style for the first time, we never had one, and to avoid further changes.

Reply:
Yeah I agree. I need to steal it for ESE ;).

BtW: can we make a repo out of this?

Reply:
Good point, do you think of a template repository or an IntelliJ settings repo (if this exists)?

Reply:
I have no idea what would be best here. Intention is just a simple way to keep this in sync between (meta) projects. Like a shared library.